crates/pallets/starknet/src/tests/block.rs

```diff
@@ -121,7 +122,7 @@ fn get_block_context_works() {
             block_context.fee_token_address
         );
         // correct vm_resource_fee_cost
-        assert_eq!(HashMap::<String, f64>::new(), block_context.vm_resource_fee_cost);
+        assert_eq!(Arc::new(HashMap::<String, f64>::new()), block_context.vm_resource_fee_cost);
```

Comment:
You can probably use `Default::default()` instead for readability

Reply:
I actually tried that before but assert wasn't able to get the type due to multiple implementations of `PartialEq`. I have now changed it to the following

```rust
let vm_resoursce_fee_cost: Arc<HashMap<String, f64>> = Default::default();
assert_eq!(vm_resoursce_fee_cost, block_context.vm_resource_fee_cost);
```